### DISCUSSION TITLE
[FF7] Steam Deck/Linux and Wine/Proton support.

### DISCUSSION THREAD

**Discussion by Uklosk** [julianxhokaxhiu/FFNx#439] [Ideas]
Hi, FFNx seems to work out of the box on Linux, at least on my Steam Deck, but it crashes when loading a savegame.

For example, this is a savegame just after the first reactor, I can load it on Windows, but on Linux it crashes and the error message redirects me to the FAQ. I can start a new game from the beginning without any issues.

It seems some minor issue that can easily be fixed, but I right know lack a proper Linux installation in order to do further testing.

Here are the save and the log files.
[save00.zip](https://github.com/julianxhokaxhiu/FFNx/files/8557494/save00.zip)
[FFNx.log](https://github.com/julianxhokaxhiu/FFNx/files/8557504/FFNx.log)

**Comment by julianxhokaxhiu** [comment]
Hello, consider updating to the latest Canary and to try again. Your version is out-of-date and no support will be given. Thank you.

**Reply by Uklosk** [reply]
Thanks for the quickly response.

I installed Satsuki Yatoshi Mod AIO package, so that is probably the version that comes with it.

I tried updating FFNx by just pasting the latest Canary over the installation but the log keeps printing the older one, I will take a deeper look after work today (it's 10AM in Spain right now) and report back.

**Reply by julianxhokaxhiu** [reply]
Remember that I am not able to provide official support for Linux nor the Steam Deck, but I'll be happy to document ever workaround required to make it work. This is because this is a project targeting the Windows binary first, and second because I do not own a Steam Deck myself so it would be impossible to provide a good ground when you don't own the device in the first place.

Thank you in advance.

**Comment by Uklosk** [comment]
I tried the same save in a laptop with a clean Arch Linux installation and I get the same error with the latest Canary, here is the log:

[FFNx.log](https://github.com/julianxhokaxhiu/FFNx/files/8577226/FFNx.log)

I understand that Linux is out of the scope of the project, so I will gladly try and debug it myself now that I have a proper environment available.

I will post any findings here.

Thanks and regards.
